cypress/FakeSVapi.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const app = express();
+const cors = require('cors');
+const port = 8080;
```

Comment:
FYI Cypress can intercept network requests and provide stubbed responses on it's own, if all of your requests are made client-side - so you might not need to do this.

https://docs.cypress.io/guides/guides/network-requests.html#Requests

Reply:
I might be wrong, but I think they only intercept XHR and not fetch requests T_T

Reply:
Ahh yeah, good spot! Thanks Kat - you're right, `fetch` isn't not supported by the looks of it - weird, I would have expected them to support that.

https://github.com/cypress-io/cypress/issues/95